### DISCUSSION TITLE
Change TreeView's item height

### DISCUSSION THREAD

**Discussion by danilwhale** [AvaloniaUI/Avalonia#13898] [Q&A]
I have defined TreeView like this:
```xaml
        <TreeView Grid.Row="1" ItemsSource="{Binding Items}">
            <TreeView.ItemTemplate>
                <TreeDataTemplate ItemsSource="{Binding SubNodes}">
                    <Grid ColumnDefinitions="32, *">
                        <Image Grid.Column="0"></Image>
                        <TextBlock Grid.Column="1" Text="{Binding Name}" FontSize="12"></TextBlock>
                    </Grid>
                </TreeDataTemplate>
            </TreeView.ItemTemplate>
        </TreeView>
```
and I get this:
![image](https://github.com/AvaloniaUI/Avalonia/assets/61111955/c8f8fb01-40f9-4092-9552-d36aeffc5d26)
Is there any way to make it lower in height? I dont like very big controls

**Comment by timunie** [comment]
change this setter or override the dynamic resource: 

https://github.com/AvaloniaUI/Avalonia/blob/0a6987b0855bc7cd0b371571bb0234dc779da181/src/Avalonia.Themes.Fluent/Controls/TreeViewItem.xaml#L64-L64
